server/api/store.go

```diff
@@ -28,12 +28,14 @@ import (
 	"github.com/unrolled/render"
 )
 
-type metaStore struct {
+// MetaStore contains meta information about a store.
+type MetaStore struct {
```

Comment:
why public heere?

Reply:
tidb_exporter need these structs to unmarshal